students/arun_nalla/session03/list_lab.py

```diff
@@ -0,0 +1,58 @@
+#!usr/bin/env python
+# Series 1
+# Create a list containing fruits and print the list
+fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print (fruit_list)
+# users input to add a fruit, add the fruit to list and print list
+response = input('Please add a fruit of your interest\n>>> ').title()
+fruit_list.append(response)
+print (fruit_list)
+number = int(input('Please enter a number either greater than 0 but should be less than the number of items to know the place of the fruit in the list: '))
+if number ==0 or number>=len(fruit_list):
```

Comment:
I suppose `len(fruit_list)` for `number` might be a valid number, so the second condition should be `number > len(fruit_list)`...?